zerver/lib/export.py

```diff
@@ -2707,19 +2708,10 @@ def get_id_list_gently_from_database(*, base_query: Any, id_field: str) -> list[
     return all_ids
 
 
-def chunkify(lst: list[int], chunk_size: int) -> list[list[int]]:
-    # chunkify([1,2,3,4,5], 2) == [[1,2], [3,4], [5]]
-    result = []
-    i = 0
-    while True:
-        chunk = lst[i : i + chunk_size]
-        if len(chunk) == 0:
-            break
-        else:
-            result.append(chunk)
-            i += chunk_size
-
-    return result
+# We only require Python 3.10, which does not include itertools.batched; include our own equivalent
```

Comment:
I wonder if we should establish a syntax of the form `TODO/Python3.10` for this sort of comment to make them easier to find in auditing. Don't think it really matters given how short the code is anyway.

Reply:
The `itertools.batched` code _is_ written in C, so will be faster than this Python code.  But, yeah I included the version number explicitly so that it might turn up in `grep`s when we bump the version.